package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "osrm",
-  "version": "5.11.0",
+  "version": "5.11.0-latest.1",
   "private": false,
```

Comment:
How about setting versions on `master` to `0.0.0` (both package.json and CMakeLists.txt) always, and only modifying the version field on release branches?